pytorch_lightning/core/lightning.py

```diff
@@ -579,8 +579,8 @@ def validation_epoch_end(
             Dict or OrderedDict.
             May have the following optional keys:
 
-            - progress_bar (dict for progress bar display; only tensors)
-            - log (dict of metrics to add to logger; only tensors).
+            - progress_bar (dict for progress bar display; either tensors or scalars)
```

Comment:
I would write "either scalar tensors or Python scalars" to make it clear what kind of scalar we mean here.